reinvent_plugins/components/RDKit/comp_pmi.py

```diff
@@ -40,7 +40,9 @@ def __call__(self, mols: List[Chem.Mol]) -> np.array:
         for mol in mols:
             try:
```

Comment:
There is really nothing in the try block that raises ValueError as far as I can tell.  AddHs does not raise exceptions an neither does CalcNPR1/2 (returns 0.0 on error which is useless to detect if it had failed).  Can you remove the try block and only check on failure of Chem.EmbedMolecule, please?  Many thanks.

Reply:
Ok yeah all done!